skyportal/handlers/api/classification.py

```diff
@@ -28,12 +28,9 @@ def get(self, classification_id):
               application/json:
                 schema: Error
         """
-        classification = Classification.get_if_readable_by(
-            classification_id, self.current_user
+        classification = Classification.get_if_accessible_by(
+            classification_id, self.current_user, raise_if_none=True
```

Comment:
How does this `raise_if_none` tie in with `self.error`?

Reply:
I think it more or less produces the same effect. It causes an AccessError to raise which looks like: 

```
class AccessError(tornado.web.HTTPError):
    def __init__(self, reason, status_code=400):
        tornado.web.HTTPError.__init__(self, reason=reason, status_code=400)

    def __str__(self):
        return self.reason
```

I confirmed by looking at the logs that this produces the same log state (no python Exception in the logs) as calling self.error explicitly, e.g., 

[20:41:34 basehandler] Error response returned by [/api/sources/14gqr]: [Insufficient permissions.]

is the only log entry produced for an API call that triggers `raise_if_none`.

Reply:
Ah, OK, it's a bit confusing, because we try very hard not to raise exceptions from API calls.

Perhaps we can rename this to give a better sense of what's happening behind the scenes.  `error_if_none`?